### PR TITLE
Fixes #25170 - Update admin/pass param to initial_param.

### LINKED ISSUE #406: Wrong admin username and password for sqlite database
If sqlite database is in use, `admin_username` and `admin_password` are not used.

### PR DESCRIPTION
**Pull request by chris1984**
Fixes #406

### CODE REVIEW

**Comment by ekohl** [comment]
Perhaps we should open a redmine issue for this as well for easier tracking in the foreman release log. This will impact installation manuals for example.

**Comment by ekohl** [comment]
@ares IMHO the parameter name should be self explanatory. With the amount of parameters we have we simply can't expect people to read them all. We already do so for the organization and location which is also part of the seeding defaults. This makes it consistent. In RH we also had downstream complaints about it besides the already linked issue.

**Comment by ares** [comment]
Ok, no problems with that if we add migration and improve the parameter description. It's just that everyone who has the.installation scripted will need to update the script as the initial password is likely being set.

**Comment by chris1984** [comment]
Related installer prs:

Katello:
https://github.com/Katello/katello-installer/pull/699

Foreman:
https://github.com/theforeman/foreman-installer/pull/296

File: spec/classes/foreman_spec.rb [hunk 1]
```diff
@@ -43,92 +43,92 @@ class { 'foreman::providers':
...
-          :logging_level             => 'info',
-          :loggers                   => {},
-          :email_delivery_method     => 'sendmail',
-          :email_smtp_address        => 'smtp.example.com',
-          :email_smtp_port           => 25,
-          :email_smtp_domain         => 'example.com',
-          :email_smtp_authentication => 'none',
-          :email_smtp_user_name      => 'root',
-          :email_smtp_password       => 'secret',
+          :foreman_url                       => 'http://localhost',
```
Comment (ekohl) [inline spec/classes/foreman_spec.rb]: This whitespace re-alignment isn't needed

File: spec/classes/foreman_database_spec.rb [hunk 2]
```diff
@@ -63,8 +63,8 @@
       describe 'with seed parameters' do
```
Comment (ekohl) [inline spec/classes/foreman_database_spec.rb]: Maybe this should be exanded to include all `initial_*` parameters to verify they're set correctly.

**Comment by ekohl** [comment]
This will also need a change in forklift. Grep for foreman-admin-password and anything that applies nightly needs an update. It is the last change I want to get in before 10.0 and before branching. Usually I prefer to avoid such a breaking change so late, but I really want to avoid a major version bump for a while.

cc @tbrisker

**Comment by tbrisker** [comment]
@ekohl how close is this to being merged? and does this mean we're blocked on branching the installer?  How important is getting this change in for 1.20?
seeing that this has impacts on installer, forklift and possibly breaking user's scripting, i'm inclined to suggesting we push this off to the next version, but i'm willing to be persuaded otherwise.

**Comment by chris1984** [comment]
@tbrisker I can get all the prs updated tonight, first thing tomorrow. It would be nice to get this into 6.5

**Comment by ekohl** [comment]
@chris1984 in the release meeting we decided it was too late in the branching cycle to pull this in. I'm going to do a 10.0.0 release of this module and soon after we can merge this starting the 11.0.0 cycle. For downstream we can find a solution, but don't need to discuss this in upstream.

**Comment by ekohl** [comment]
@chris1984 could you rebase this? Now that the installer merge is complete, I'd like to get this in.

**Pull request closed without merging**